packages/xo-server/src/api/pool.mjs

```diff
@@ -29,7 +30,8 @@ export async function set({
     auto_poweron !== undefined && pool.update_other_config('auto_poweron', String(auto_poweron)),
     nameDescription !== undefined && pool.set_name_description(nameDescription),
     nameLabel !== undefined && pool.set_name_label(nameLabel),
-    migrationNetwork !== undefined && pool.update_other_config('xo:migrationNetwork', migrationNetwork),
+    migrationCompression !== undefined && pool.set_migration_compression(migrationCompression),
+    migrationNetwork !== undefined && pool.update_other_config('xo:migrationNetwork', String(migrationNetwork)),
```

Comment:
Why did you modify this line?

Reply:
Wrong line, didn't see it before adding to the commit ! Reverting it right now